### PR TITLE
Task/missing options and filters

### PR DESCRIPTION
**Pull request by alonisser**

### CODE REVIEW

File: configuration/views.py [hunk 1]
```diff
@@ -101,7 +102,10 @@ def uncon_studies_form(self, request, **kwargs):
         available_stimulus_modality_type = UnConModalityType.objects.all()
         available_stimulus_category_type = UnConStimulusCategory.objects.all()
         available_stimulus_sub_category_type = UnConStimulusSubCategory.objects.all()
+        available_outcomes_type = UnConFinding.objects.values_list("outcome", flat=True).distinct()
```
Comment (dani-efr) [inline configuration/views.py:105]: גם פה צריך לשנות אחר כך נכון?

File: uncontrast_studies/processors/parameters_distribution_free_queries.py [hunk 2]
```diff
@@ -92,6 +94,21 @@ def get_queryset(self):
         if len(self.suppression_methods_types):
             queryset = queryset.filter(suppression_methods__type__id__in=self.suppression_methods_types)
 
+        if len(self.outcome_types):
+            # TODO: fix this later to use the type
```
Comment (dani-efr) [inline uncontrast_studies/processors/parameters_distribution_free_queries.py:98]: which one of us will fix this? I don't think I know how to

**Review by dani-efr — APPROVED** [review]

File: uncontrast_studies/processors/parameters_distribution_free_queries.py (hunk 2 above)
Comment (alonisser) [reply uncontrast_studies/processors/parameters_distribution_free_queries.py:98]: I would after you introduce a type

File: configuration/views.py (hunk 1 above)
Comment (alonisser) [reply configuration/views.py:105]: true

**Pull request merged**